### ISSUE TITLE
[Bug] Wrong auto complete

### ISSUE THREAD

**Issue by Seng-Jik** [ionide/ionide-vscode-fsharp#1773]
### Describe the bug
![image](https://user-images.githubusercontent.com/11296891/186444244-54a33b87-f9ab-4393-8af7-374aa00f1fb5.png)

https://user-images.githubusercontent.com/11296891/186443827-35c65636-835f-4a20-9f2d-25d0c9e50e11.mp4

### Steps to reproduce
1. Create a 'fsx' file and open it.
2. Just type a `Map.`, `Set.`, `List.`, some times it happen.
3. In bigger F# project, it will more frequently.

### Expected behaviour
Just popup a correct auto complete box with correct options.

### Machine info
 - OS: Windows 11
 - .NET SDK version: 6.0.302
 - Ionide version: v7.0.1

### Additional context

I am in China and I'm using a unstable VPN.

**Comment by Seng-Jik** [comment]
```json
"FSharp.resolveNamespaces": false,
    "FSharp.keywordsAutocomplete": false,
    "FSharp.abstractClassStubGeneration": false,
    "FSharp.enableTouchBar": false,
    "FSharp.inlayHints.parameterNames": false,
    "FSharp.inlayHints.typeAnnotations": false,
    "FSharp.showExplorerOnStartup": false,
    "FSharp.dotnetRoot": "C:/Program Files/dotnet",
```

**Comment by Seng-Jik** [comment]
v6.0.6 has less chance of this error.

**Comment by baronfel** [comment]
I believe this is resolved in 7.5.0 and up.
